script.js

```diff
@@ -22,22 +23,32 @@ function addBook() {
         number_of_copies: number_of_copies
     }
     //books.push('book');
-    //console.log(`${title} = ` + 'the newbook is table');
+   
     let hasBook = false;
     let bookSave = document.getElementById('book-save');
     let bookBd = document.getElementById('book-bd');
-
+    
     for(let i = 0; i < books.length; i++) {
         let currentBook = books[i];
         if(book.id_number == currentBook.id_number) {
             hasBook = true;
+            books.push(book);
```

Comment:
we don't need to save (push) the same book again to database

Reply:
create function to show Error Status in other place and call it here

show block with error message
hide block with success message
clear input

Reply:
after you call showErrorStatus() function we return from addBook()

Reply:
done